rice-datanode/source/data_transfer_server.cc

```diff
@@ -307,3 +307,8 @@ void TransferServer::synchronize(std::function<void(TransferServer&, tcp::socket
 	dn->decrementNumXmits();
 	cv.notify_one();
 }
+
+bool TransferServer::sendStats() {
+	uint64_t free_space = fs->getFreeSpace();
+	return dn->sendStats(free_space);
```

Comment:
Since we have this method now, can we make TransferServer own xmits? And send xmits as an arg in dn->sendStats.